src/content/docs/en/reference/content-loader-reference.mdx

```diff
@@ -351,6 +351,40 @@ export function feedLoader({ url }): Loader {
 }
 ```
 
+#### `renderMarkdown`
+
+<p>
+**Type**: `(markdown: string) => Promise<RenderedContent>`
+</p>
+<Since v="5.9.0" />
+
+Renders a Markdown string to HTML, returning a `RenderedContent` object. This allows the entry to be rendered in a page using the [`render()` function and `<Content />` component](/en/guides/content-collections/#rendering-body-content). To enable this, assign the rendered content to the [`rendered`](#rendered) field of the [`DataEntry`](#dataentry) object.
```

Comment:
```suggestion
Renders a Markdown string to HTML, returning a `RenderedContent` object.

This allows allows you to render Markdown content directly within your loaders using the same Markdown processing as Astro's built-in `glob` loader and provides access to the `render()` function and `<Content />` component for [rendering body content](/en/guides/content-collections/#rendering-body-content).

To process Markdown in your loader, assign the rendered content to the [`rendered`](#rendered) field of the [`DataEntry`](#dataentry) object:
```
OK, borrowed a bit from the changeset here. What about *this*? 😅

Reply:
The final sentence still isn't right! You can process Markdown without assigning it like that – it just means users won't be able to use the `render()` feature.

Reply:
The key points are:

- if you assign an HTML string to `rendered.html`, users can use `render()` and `<Content />` to display the rendered content on a page. That HTML could come from anywhere, e.g. it may be returned from the CMS.
- with the `renderMarkdown` helper, you can render markdown to HTML. What you do with that HTML is up to you.
- ...therefore, you can use the helper to render the markdown to HTML and then assign it to `rendered` and users will be able to use `render()` and `<Content />` to display the rendered content on a page

Reply:
OK. Could you do it as a suggestion so it's clear?

Reply:
Yes, edited my suggestion above to be MOST RECENT.

Reply:
We did it Joe!